internal/v1/api.yaml

```diff
@@ -595,7 +832,7 @@ paths:
     get:
       summary: get composes associated with a blueprint
       description: "get a collection of composes associated to a blueprint, allows for filtering by version"
-      operationId: getBlueprintComposes
+      operationId: getBlueprintComposesExperimental
```

Comment:
Sharing the id is not supported? That would enable you to drop the wrapper functions.